lib/network_ssl/network_ssl.c

```diff
@@ -25,7 +25,7 @@
 #endif
 
 /* SSL context in which to create connections. */
-SSL_CTX * ctx = NULL;
+static SSL_CTX * ctx = NULL;
```

Comment:
Err, by "clang is unhappy as it is", I meant "with the previous code".  It's happy if we have `static` here.